spk/tor/src/wizard/uninstall_uifile

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "step_title": "TOR Uninstallation Note",
+        "items": [
+            {
+                "desc": "If you like to backup your configuration and ID, please backup /usr/local/tor/var BEFORE you uninstall this package!"
```

Comment:
Another way of putting it, a bit more in line with other package uninstall messages:
`Attention: The Tor configuration and ID will be removed during package uninstallation. Remember to backup /usr/local/tor/var before you uninstall this package`